src/test/java/RandomComputerPlayerTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.Test;
+
+import java.util.List;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class RandomComputerPlayerTest {
+
+    private RandomComputerPlayer randomComputerPlayer = new RandomComputerPlayer();
```

Comment:
Because RandomComputerPlayer implements Player, if you only need methods available on the interface, you can actually declare this as:
```private Player randomComputerPlayer = new RandomComputerPlayer()```

If you need methods that are not on the interface, then it could be an LSP violation... 🤔